pandas/tests/series/test_missing.py

```diff
@@ -1064,19 +1064,32 @@ def test_interp_limit(self):
         # GH 9217, make sure limit is an int and greater than 0
         methods = ['linear', 'time', 'index', 'values', 'nearest', 'zero',
                    'slinear', 'quadratic', 'cubic', 'barycentric', 'krogh',
-                   'polynomial', 'spline', 'piecewise_polynomial', None,
+                   'polynomial', 'spline', 'piecewise_polynomial',
```

Comment:
The specific value `None` is now checked in the test case `test_interp_invalid_method` below.